packages/opentelemetry-exporter-collector/src/platform/node/CollectorExporter.ts

```diff
@@ -29,6 +29,8 @@ import { toCollectorExportTraceServiceRequest } from '../../transform';
 import { GRPCQueueItem, TraceServiceClient } from './types';
 import { removeProtocol } from './util';
 
+const DEFAULT_COLLECTOR_URL = 'http://localhost:55678';
```

Comment:
I think we should change this URL to `localhost:55678` and remove [removeProtocol](https://github.com/open-telemetry/opentelemetry-js/blob/e9b2cf9aeb1daf5ffbab800681bfe1cafc636576/packages/opentelemetry-exporter-collector/src/platform/node/CollectorExporter.ts#L66) logic from Node specific `CollectorExporter` module.

Reply:
I would change the default url, but would live the `removeProtocol` to simply make it error proof for users